views/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+    
+    <head>
+        <title>Geosci</title>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>  
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+        <meta name="description" content="Open-source resources to facilitate education, research and collaboration in the geoscience community">
+        <link rel="stylesheet" href="/assets/css/bootstrap.css">
+        <link rel="stylesheet" href="/assets/css/ionicons.css">
+        <link rel="stylesheet" href="/assets/css/style.css">
+    </head>
+
+    <body>
+        {% include _templateFolder+'nav.html' %}
+            
+        <div class="container">
+            <div class="row">
+                <div class="col-xs-12 col-sm-6 col-md-4 col-lg-4">
+                    <div class="site">
+                        <img class="site-img" src="http://www.tapir.caltech.edu/~teviet/Waves/dipole.gif" />
+                        <a href="http://google.ca">
+                            <span class="size-32">
+                                <i class="ion-ios-nutrition site-icon"></i>
+                            </span>
+                        </a>
+                        <p class="site-title"><a href="http://google.ca" class="">Sample Site</a></p>
+                    </div>
+                </div>
+                <div class="col-xs-12 col-sm-6 col-md-4 col-lg-4">
+                    <div class="site">
+                        <img class="site-img" src="http://www.tapir.caltech.edu/~teviet/Waves/dipole.gif" />
+                        <a href="http://google.ca">
+                            <span class="size-32">
+                                <i class="ion-ios-nutrition site-icon"></i>
```

Comment:
can you use font awesome icons instead (https://fortawesome.github.io/Font-Awesome/get-started/), for example. the book: http://fortawesome.github.io/Font-Awesome/icon/book/